blockchains/cardano/cardano_worker.js

```diff
@@ -170,7 +170,7 @@ class CardanoWorker extends BaseWorker {
   }
 
   async work() {
-    const fromBlock = this.lastExportedBlock + 1;
+    const fromBlock = this.lastBufferedBlock + 1;
```

Comment:
Not a fan of all those renames. You resorted to renaming the 'last exported block' to 'last buffered block' to make it comply more easily with your change. But:
1. All those other workers don't know what 'buffered block' is
2. I would argue that 'last exported block' is cleaner in terms of semantics. The last block which got into Kafka. What is 'last buffered block'?